src/SprykerEco/Zed/PunchoutCatalogs/Persistence/PunchoutCatalogsEntityManager.php

```diff
@@ -62,6 +62,8 @@ public function updatePunchoutCatalogConnection(PunchoutCatalogConnectionTransfe
                 $punchoutCatalogConnectionEntity
             );
 
-        return (bool)$punchoutCatalogConnectionEntity->save();
+        $punchoutCatalogConnectionEntity->save();
```

Comment:
Why not return `$vaultDepositEntity->save() > 0`?
Or why you even reverted this?
It looks like a behavior change.

Reply:
Yep, it's behavior change.
Because when you execute `updateSmth` method you do not expect that it'll return false even in the success case: when data in DB already match to what you are trying to update it